UI/multitrack-video-output.hpp

```diff
@@ -52,9 +52,13 @@ struct MultitrackVideoOutput {
 
 private:
 	struct OBSOutputObjects {
-		OBSOutputAutoRelease output_;
 		std::shared_ptr<obs_encoder_group_t> video_encoder_group_;
 		std::vector<OBSEncoderAutoRelease> audio_encoders_;
+		/* Outputs do not hold on to their encoders,
+		 * so the output needs to be stopped/released
+		 * before the encoders are released
+		 */
+		OBSOutputAutoRelease output_;
```

Comment:
I'm not really a fan of how this relies on implicit c++ delete behavior to keep libobs from crashing. I'm looking into if it's possible to solve the issue from the libobs side.